desktop-widgets/command_edit.h

```diff
@@ -40,6 +35,11 @@ class EditBase : public Base {
 	EditBase(const QVector<dive *> &dives, T newValue, T oldValue);
 
 protected:
+	T value; // Value to be set
+	T old; // Previous value
+	void undo() override;
+	void redo() override;
+
```

Comment:
why are these now protected? Is this a logical part of this commit?

Reply:
Because the dive-site editing does some special stuff like deleting non-existing dive sites and therefore "hijacks" the `undo()` call. To be able to call the `undo()` function of the parent class that has to be made protected instead of private. Of course, I could change the history to make them protected from the start...